avo-hq/avo
# feature: multiple action alerts

# Description


Fixes https://github.com/avo-hq/avo/issues/881 and adds the ability to respond with multiple types of alerts using the `succeed`, `fail`, `inform`, and `warn` methods.

<img width="537" alt="alert-responses" src="https://user-images.githubusercontent.com/1334409/166519804-fe5e9310-4b33-4780-8eec-3413c33251f4.png">

# Checklist:



## Manual review steps


1. Go to users
2. Click on Actions
3. Click on Dummy
4. Click on Run
5. Expect to see four types of alerts (info, error, success, warning) displayed

Manual reviewer: please leave a comment with output from the test if that's the case.
